packages/Webkul/Shop/src/Resources/views/components/layouts/header/mobile/index.blade.php

```diff
@@ -34,8 +34,60 @@
                 </x-slot:header>
 
                 <x-slot:content>
+                    {{-- Localization & Currency Section --}}
+                    <div class="flex w-full justify-between items-center mb-[15px] gap-x-[20px]">
+
+                        <x-shop::dropdown position="bottom-left">
```

Comment:
Some classes are misplaced. Please rearrange them in the following **order**

**[** custom classes, width and height properties, display property, gap property, alignment property, margin and padding, pseudo-classes, and media property. **]**